kusto/conn.go

```diff
@@ -204,9 +204,9 @@ func (c *conn) execute(ctx context.Context, execType int, db string, query Stmt,
 }
 
 func (c *conn) Close() error {
-	if closer, ok := c.auth.(io.Closer); ok {
+	/*if closer, ok := c.auth.(io.Closer); ok {
```

Comment:
?

Reply:
Since the auth for conn struct is removed so we had to remove this too.